catalog/models.py

```diff
@@ -250,53 +250,27 @@ def exclude_by_products(self, products: Iterable[AbstractProduct]):
             .distinct()
         )
 
-    # @todo #SE747:30m  Rm `get_group_tags_pairs`.
-    #  Use `get_grouped_tags -> Dict[Group, List[Tag]]` instead.
-    #  Tags pairs already lead us to errors with groups order.
-    def get_group_tags_pairs(self) -> List[Tuple[TagGroup, List['Tag']]]:
+    def get_group_tags_pairs(self) -> Dict[TagGroup, List['Tag']]:
```

Comment:
let's rename it to `get_grouped_tags` or even better `group_tags`.
It's not pairs anymore

Reply:
i will create todo for it